lib/bot/chat_bot/chat/session/server.ex

```diff
@@ -92,7 +92,12 @@ defmodule Telegram.Bot.ChatBot.Chat.Session.Server do
   @impl GenServer
   def handle_cast({:handle_update, update}, %State{} = state) do
     res = state.chatbot_behaviour.handle_update(update, state.token, state.bot_state)
-    handle_callback_result(res, state)
+
+    if Utils.transient_chat?(update) do
+      handle_callback_result({:stop, elem(res, 1)}, state)
```

Comment:
this feels a little wonky but it was the simplest way to make the process handling inline query updates shut down after they handle the request